editorial_material/why_livecoms.tex

```diff
@@ -98,6 +99,7 @@ \section{LiveCoMS combines the concepts of preprints, perpetual reviews, and ove
 First, for minor revisions, LiveCoMS publications can link to the preprint server or other repository where ALL versions of a paper will be placed, so readers visiting the published version of the article will be able to click through to the most up-to-date version.
 Second, significant revisions are expected to be made on a fairly regularly basis (perhaps yearly to every couple of years) and these can be published again in LiveCoMS, allowing authors to obtain additional publication credit for work they have done updating and maintaining these. 
 
+%MRS: should this section below go earlier?
```

Comment:
Maybe. What's your thought process? Mine was that this was part of wrap-up -- first we explain what the problem is that needs solving, how it applies primarily to particular types of articles, what our plan is for solving it, etc., then we explain that we're going to primarily focus on articles which HAVE that problem. But maybe there is a better way of saying it?